README.md

```diff
@@ -11,7 +11,7 @@ allowing taproot multisig and locktime-based spending conditions.
 
 Do a reliable hodling (with inheritance options), corporate/organization, & 
 current accounts - or instant Lightning payments. Work with single- and 
-multisig setups, based on hardware, air-gaped, cold and server-side hot key 
+multisig setups, based on hardware, air-gapped, cold and server-side hot key 
```

Comment:
Thank you! I actually think it should be `air gapped`: https://github.com/mycitadel/mycitadel-desktop/issues/45

Anyway, this had to be fixed in the `v2` branch

```suggestion
multisig setups, based on hardware, air gapped, cold and server-side hot key 
```